marketplace-kit-archive.js

```diff
@@ -1,56 +1,80 @@
 #!/usr/bin/env node
 
-const program = require('commander'),
-  fs = require('fs'),
-  shell = require('shelljs'),
-  archiver = require('archiver'),
-  logger = require('./lib/kit').logger,
-  validate = require('./lib/validators'),
-  version = require('./package.json').version;
+const program = require("commander"),
```

Comment:
can you install prettier-eslint for it to respect .eslintrc file settings when prettifying files? our editors will fight otherwise on every commit :0